client/components/Navbar.js

```diff
@@ -30,23 +30,45 @@ const NavBar = props => {
     let firstLetter = petType.type[0].toUpperCase();
     let restOfWord = petType.type.slice(1);
     let typeOfPet = firstLetter + restOfWord;
-    return <Link to={`/pets/${petType.type}`} key={petType.id}>{typeOfPet} </Link>
+    return(
+      <li key={petType.id}>
+        <Link to={`/pets/${petType.type}`} >{typeOfPet} </Link>
+      </li>
+    ) 
   })
-
+  
   return(
     <div>
+      <div id="top-header">
+        <div>
+          {/* <img src={require("../assets/images/logo.png")} alt="logo" /> */}
```

Comment:
Is this commented out code that we can remove?

Reply:
Yes. Will remove. Serge may try adding the logo in later, but it will be on a separate branch.